src/OpenTelemetry.Instrumentation.AWSLambda/CHANGELOG.md

```diff
@@ -11,6 +11,8 @@
   ([#1295](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1295))
 * BREAKING: Target `net6.0` instead of `netstandard2.0`
   ([#1545](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1545))
+* Fix native AoT warnings
```

Comment:
Warning fixes are not relevant for end users, but if the warnings were relevant, would it be true to say "Add support for AoT compilation"?

Reply:
```suggestion
* Add support for native AoT.
```